src/x86/cpuid.c

```diff
@@ -20,8 +20,9 @@
 #include "uarch.h"
 #include "freq/freq.h"
 
-#define CPU_VENDOR_INTEL_STRING "GenuineIntel"
-#define CPU_VENDOR_AMD_STRING   "AuthenticAMD"
+#define CPU_VENDOR_INTEL_STRING         "GenuineIntel"
```

Comment:
Please leave only one space between CPU_VENDOR_TRANSMETA_STRING and "GenuineTMx86", then adjust AMD and Intel accordingly.